app/commands/xp.js

```diff
@@ -341,8 +341,8 @@ module.exports = Base => class extends Base {
             this.tellPlayer(playerName, `No pending experience point offers.`, 'red');
             return;
         }
-        let offeringPlayer = this.giveOffers[playerName][0];
-    
+
+        let offeringPlayer = this.giveOffers[playerName][0];   
```

Comment:
Why is this edited?

Reply:
i was looking through to see what i'd done in the past here and decided i didn't like the way stuff was spaced